docs/DeveloperGuide.adoc

```diff
@@ -295,20 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |know the total amount I have spent |plan my finances for the remaining days of the month.
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |greedy user |have reminders when I am about to hit the limits I set for my expenditure |plan my budget better for the weeks ahead.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |meticulous user |sort the food items based on categories |see which items are the best or worst based on certain categories.
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |forgetful user |track my expenses for the day using the app |know how much I have already spend in the day and plan for the remaining meals of the day.
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |new user |view more information about command |learn how to use the app and its features.
 
-_{More to be added}_
+|`* * *` |user |add a food item by specifying the item, price, description and category |update the food items that are available to me if I find new food item options within NUS.
```

Comment:
User stories are well done with clear format, project direction and benefits. But, maybe you can sort user stories according to alphabetical order of users in addition to priority?